airflow/providers/amazon/aws/hooks/redshift_data.py

```diff
@@ -0,0 +1,152 @@
+        :param sql: the SQL statement text to run
+        :type sql: str
+        :param db_user: the database user name
+        :type db_user: str
+        :param parameters: the parameters for the SQL statement
+        :type parameters: list
+        :param secret_arn: the name or ARN of the secret that enables db access
+        :type secret_arn: str
+        :param statement_name: the name of the SQL statement
+        :type statement_name: str
+        :param with_event: indicates whether to send an event to EventBridge
+        :type with_event: bool
+
+        """
+        """only provide parameter argument if it is valid"""
+        if parameters:
+            response = self.get_conn().execute_statement(
+                ClusterIdentifier=cluster_identifier,
+                Database=database,
+                Sql=sql,
+                DbUser=db_user,
+                WithEvent=with_event,
+                SecretArn=secret_arn,
+                StatementName=statement_name,
+                Parameters=parameters,
+            )
+        else:
+            response = self.get_conn().execute_statement(
+                ClusterIdentifier=cluster_identifier,
+                Database=database,
+                Sql=sql,
+                DbUser=db_user,
+                WithEvent=with_event,
+                SecretArn=secret_arn,
+                StatementName=statement_name,
+            )
+        return response['Id'] if response['Id'] else None
+
+    def describe_statement(
+        self,
+        id: str,
+    ):
+        """
+        Describes the details about a specific instance when a query was run
+        by the Amazon Redshift Data API
+
+        :param id: the identifier of the SQL statement to describe.
+        :type id: str
+
+        """
+        response = self.get_conn().describe_statement(
+            Id=id,
+        )
+        return response['Status']
+
+    def get_statement_result(
+        self,
+        id: str,
+        next_token: Optional[str] = "",
+    ):
+        """
+        Fetches the temporarily cached result of an SQL statement, a token is
+        returned to page through the statement results
+
+        :param id: the identifier of the SQL statement to describe.
+        :type id: str
+        :param next_token: a value that indicates the starting point for the next set of response records
+        :type next_token: str
+
+        """
+        response = self.get_conn().get_statement_result(
+            Id=id,
+            NextToken=next_token,
+        )
+        return response
+
+    def cancel_statement(
+        self,
+        id: str,
+    ):
+        """
+        Cancels a running query, to be canceled, a query must be running
+
+        :param id: the identifier of the SQL statement to describe.
+        :type id: str
+
+        """
+        response = self.get_conn().cancel_statement(
+            Id=id,
+        )
+        return response['Status'] if response['Status'] else None
```

Comment:
```suggestion
        return response['Status']
```
Do you suspect that it could return None?

Reply:
Done